scripts/run_max_ent.py

```diff
@@ -0,0 +1,282 @@
+"""Script running the MaxEnt IRL
+algorithm on a trees data set.
+Usage:
+    python scripts/run_max_ent.py data=$PWD/path_to_trees.csv n_action=5
+Note:
+    The data path must be absolute not relative.
+"""
+import dataclasses
+import logging
+from typing import Any, Dict, Optional
+
+import anytree
+import pandas as pd
+import numpy as np
+
+import phirl.api as ph
+import phirl.hydra_utils as hy
+
+
+# data = pd.read_csv("tree_df.csv", delimiter=",")
+# state_combinations = [list(i) for i in itertools.product([0, 1], repeat=5)]
+# n_states = 32
+# n_action = 5
+
+
+def get_all_trajectory_states_features(trees):
+    """
+    Args:
+        trees: a dataframe with tree information with column names: [Patient_ID, Tree_ID, Node_ID, Mutation_ID, Parent_ID]
```

Comment:
I thought `trees` is actually a dictionary mapping the tree's ID to the root node.

Reply:
Thank you Pawel. I was describing the original dataset by mistake and I have corrected it.